lib/ui/popup.js

```diff
@@ -82,20 +77,15 @@ Popup.prototype = {
     this.el.dispatchEvent(newEvent);
     this.el.addEventListener('popuphiding', this.beforePopupHide);
   },
-  onResultsMouseOver: function(evt) {
-    if (this.ignoreNextMouseOver) {
-      this.ignoreNextMouseOver = false;
-      return;
-    }
-
-    // Throttle mouseover events, which fire rapidly if the user quickly moves
-    // the mouse across different DOM elements in the results list.
-    if (this.mouseOverTimeout) {
+  onResultsMouseMove: function(evt) {
+    // Throttle mousemove events, which fire rapidly as the user moves
+    // the mouse.
+    if (this.mouseMoveTimeout) {
       return;
     }
-    this.mouseOverTimeout = this.win.setTimeout(() => {
-      this.win.clearTimeout(this.mouseOverTimeout);
-      this.mouseOverTimeout = null;
+    this.mouseMoveTimeout = this.win.setTimeout(() => {
+      this.win.clearTimeout(this.mouseMoveTimeout);
+      this.mouseMoveTimeout = null;
     }, 15);
```

Comment:
`1000/60` would be the technically correct value for this, but close enough `^.^`